### ISSUE TITLE
Flamneshot doesn't start "qt5ct: using qt5ct plugin"

### ISSUE THREAD

**Issue by Zaldor** [flameshot-org/flameshot#687]
**Flameshot version**
Flameshot 0.5.1-1(Debian)
Compiled with QT 5.9.4

**Describe the bug**
flameshot doesn't autostart, on login the icontray doesn't show but if invoked with specific commands works, if just 'flameshot' is executed on terminal this message is shown:

>  qt5ct: using qt5ct plugin
and flameshot icon in iconstray doesn't show

**To Reproduce**
I believe it may have been caused by firejail, since it was working fine before installing it, and it actually works after restart if excuted with firejail as >firejail flameshot.
not sure because before reboot big update was also done.

**Expected behavior**
Flameshot runs at login and flameshot icon shows in the icontray

**System Information**
Linux Mint 19.3 Cinnamon -  5.3.0-40-generic #32~18.04.1-Ubuntu SMP Mon Feb 3 14:05:59 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Comment by neuronetio** [comment]
I have the same problem with 0.6.0 and Qt 5.9.5 on Kubuntu (KDE)

**Comment by neuronetio** [comment]
sorry it is running but kind of hidden
![image](https://user-images.githubusercontent.com/25027696/75356306-12231980-58b0-11ea-8751-d46f7ecd7312.png)

**Comment by thorstaj** [comment]
I think I have a solution...
* Flameshot wouldn't autostart for me either, and it wouldn't  start manually if my computer booted up with the flameshot 'autostart' capability enabled.
* Once I disabled autostart, flameshot would start just fine.
* I added a 5 second delay to autostart, and now flameshot autostarts just fine.
Hopefully this helps track down the issue....
[Linux Mint 19.3 Cinnamon 4.4.8]
